src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
+    const output = {};
+
+    // Append hidden element to body
+    const screenSizer = document.createElement('div');
+    screenSizer.classList.add('c-screen-sizer');
+
+    document.body.appendChild(screenSizer);
+
+    // It should have a 'content' property containing the breakpoints
+    const breakpoints = window.getComputedStyle(document.querySelector('.c-screen-sizer'))
+        .getPropertyValue('content')
+        .replace(/["']/g, '')
+        .split(',');
+        // Gives a list of breakpoints in the form ['narrow:414px', ...etc]
+
+    // When there is no content, at this stage breakpoints should be ['']
+    if (breakpoints.length === 1 && breakpoints[0] === '') {
+        return output;
+    }
+
+    return breakpoints.reduce((prev, current) => {
+        // `current` is of the form 'narrow:414px'
+        const [breakpointName, breakpointValue] = current.split(':');
+        prev[breakpointName] = breakpointValue; // <- the initial value is used for the first iteration
+        // The object, e.g., { 'narrow': '414px' } is returned to be used as `prev` in the next iteration
+        return prev;
+    }, output); // <- initial value
+};
+
+export const getCurrentScreenWidth = () => {
+    const currentWidth = window.innerWidth;
+
+    const breakpoints = getBreakpoints();
+
+    // Order the breakpoints from widest to narrowest,
+    // takes the form [['narrow', '414px'], [...etc]]
+    const bps = [];
+    Object.keys(breakpoints).forEach(key => {
+        bps.unshift([key, breakpoints[key]]);
+    });
+
+    for (let i = 0; i < bps.length; i++) {
+        // Loops through the breakpoints (in descending order)
+        // returning the first one that is narrower than currentWidth.
+
+        const breakpointWidth = parseInt(bps[i][1], 10); // This also strips the 'px' from the string
+
+        if (i === bps.length - 1) {
```

Comment:
`if (i === bps.length - 1 || currentWidth > breakpointWidth)` Should do the same? that way it's all on one line :)